src/tests/routes/protectedRoute.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { AuthenticatedRoute } from 'routes/protectedRoute';
+import Home from 'containers/Home';
+import { AuthProvider } from 'contexts/auth';
+import { Router } from 'react-router-dom';
+import { createMemoryHistory } from 'history';
+
+describe('When AuthenticaedRoute component is mounted', () => {
```

Comment:
Minor thing but I think we should use 
```suggestion
describe('when AuthenticaedRoute component is mounted', () => {
```